main/src/ca/uwaterloo/flix/language/phase/jvm/GenExpression.scala

```diff
@@ -1131,6 +1131,12 @@ object GenExpression {
       addSourceLine(visitor, loc)
       visitor.visitLdcInsn(s)
 
+    case Ast.Constant.Regex(patt) =>
```

Comment:
This looks ok.

Reply:
The right thing for us to do is to eliminate Regex (and BigInt and BigDecimal) in `Lowering`.

We want to slim down the backend part of the compiler.

Reply:
But, how about we merge it as is, and then in a subsequent PR you or some else can take up the `Lowering` part again?

I suppose there was some issue.

Reply:
Note that anything that messes with types in lowering causes problems with type classes. So moving `BigInt` to Lowering for example would mean that we cannot have an `Add` class for `BigInt`

Reply:
:(

We can do it post monomorph then?

Reply:
I'll leave as is.